store/sqlstore/store.go

```diff
@@ -1195,3 +1198,20 @@ func (ss *SqlStore) toReserveCase(str string) string {
 
 	return fmt.Sprintf("`%s`", strings.Title(str))
 }
+
+func (ss *SqlStore) GetDBSchemaVersion() (int, error) {
+	var version int
+	if err := ss.GetMasterX().Get(&version, "SELECT Version FROM db_migrations ORDER BY Version DESC LIMIT 1"); err != nil {
```

Comment:
Is it ever possible to be "missing" a migration?

Reply:
Do you mean whether if we skip in versions? If so, yes. Actually that was one of the reasons that we picked up the migrations table. And also we show every applied migration on the system console.

Reply:
There isn't anything like that unfortunately. That is why we have a short one number in the about modal, and a detailed view of all the migrations applied in the system console to get the best of both worlds :)